### PR TITLE
[SYCL][UR][CUDA] Don't import XPTI symbols in the plugin library

### PR DESCRIPTION
**Pull request by ProGTX**
Bumps the UR tag with a fix for the CUDA adapter on Windows.

### CODE REVIEW

**Comment by ProGTX** [comment]
See https://github.com/oneapi-src/unified-runtime/pull/1911

File: sycl/cmake/modules/FetchUnifiedRuntime.cmake [hunk 1]
```diff
@@ -115,14 +115,8 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
...
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit c5d2175b5823d5b74de1e7e0d6081ab6d885bc34
-  # Merge: 99489ad4 c86beb60
-  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
-  # Date:   Wed Jul 31 14:52:26 2024 +0100
-  #     Merge pull request #1882 from przemektmalon/przemek/interop-map-memory
-  #    [Bindless][Exp] Add interop memory mapping to USM.
-  set(UNIFIED_RUNTIME_TAG c5d2175b5823d5b74de1e7e0d6081ab6d885bc34)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/ProGTX/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG c571ec40db499b36834be12b6a7ab1129aa00193)
```
Comment (omarahmed1111) [inline sycl/cmake/modules/FetchUnifiedRuntime.cmake:119]:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit ab9baf57fbc6f17576a0c63fbbfbfe38d66c440c
  # Merge: 1fef4e29 c571ec40
  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
  # Date:   Thu Aug 8 11:09:15 2024 +0100
  #     Merge pull request #1911 from ProGTX/peter/xpti-static
  #    [CUDA] Don't import XPTI symbols in the plugin library
  set(UNIFIED_RUNTIME_TAG ab9baf57fbc6f17576a0c63fbbfbfe38d66c440c)
```

**Comment by omarahmed1111** [comment]
@ProGTX This needs to be merged with latest sycl to resolve the conflict in UR bump and trigger CI

**Review by omarahmed1111 — APPROVED** [review]

**Comment by omarahmed1111** [comment]
@intel/llvm-gatekeepers Please merge when possible, Thanks!

**Pull request merged**